drivers/soundwire/bus.c

```diff
@@ -969,6 +967,10 @@ int sdw_bus_prep_clk_stop(struct sdw_bus *bus)
 	if (!simple_clk_stop) {
 		ret = sdw_bus_wait_for_clk_prep_deprep(bus,
 						       SDW_BROADCAST_DEV_NUM);
+		/*
+		 * if there are no Slave devices present and the reply is
+		 * Command_Ignored/-ENODATA, we can stop the sequence here
+		 */
```

Comment:
hm, "we can stop the sequence here" sounds to me like this isn't an error, we're done. But we do return an error and the caller `sdw_cdns_clock_stop()` also treats it as such. But since this keeps the existing behaviour I won't consider this an error, maybe just the comment wording is not perfectly clear.

Reply:
Yes, this is misleading, I will reword this in a fixup. Thanks for commenting.